lib/app.js

```diff
@@ -40,13 +40,20 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', index);
+app.use('/v1', function (req, res, next){
+  if ((req.accepts().length ===  0) || (req.accepts().length > 0 && (req.accepts()[0] !== 'text/plain'))) {
+      res.status(406).send('Not acceptable!');
```

Comment:
I don't take care about it previously but the returned message is in text format?